search_gov_crawler/search_gov_spiders/helpers/domain_spider.py

```diff
@@ -0,0 +1,112 @@
+import json
+import re
+from pathlib import Path
+from typing import Any, Optional
+
+from scrapy.http import Response
+from scrapy.linkextractors import LinkExtractor
+
+from search_gov_crawler.search_gov_spiders.items import SearchGovSpidersItem
```

Comment:
The absolute imports did not work for me, but using the relative imports (i.e from ..items import SearchGovSpidersItem) did